lib/actions/api.js

```diff
@@ -1015,30 +571,21 @@ const findGeometryForPatternError = createAction(
 )
 
 export function findGeometryForPattern(params) {
-  return function (dispatch, getState) {
-    const state = getState()
-    const route = state?.otp?.transitIndex?.routes?.[params.routeId]
-    // If a route was fetched from GraphQL, it already has geometry
-    if (route && route.v2) {
-      return
-    }
-
-    const { patternId, routeId } = params
-    return dispatch(
-      createQueryAction(
-        `index/patterns/${params.patternId}/geometry`,
-        findGeometryForPatternResponse,
-        findGeometryForPatternError,
-        {
-          rewritePayload: (payload) => ({
-            geometry: payload,
-            patternId,
-            routeId
-          })
+  return createQueryAction(
+    `index/patterns/${params.patternId}/geometry`,
+    findGeometryForPatternResponse,
+    findGeometryForPatternError,
```

Comment:
I agree this could use work, but I think our energy here should be more focused on a full graphQL implementation can remove the need for any of these actions at all (and rather be one single action that can request any variable needed)